weave/flow/eval.py

```diff
@@ -265,3 +269,12 @@ def is_valid_model(model: Any) -> bool:
             and is_op(model.predict)
         )
     )
+
+
+T = TypeVar("T")
+
+
+def repeated_iterable(iterable: Iterable[T], n: int) -> Iterable[T]:
+    for val in iterable:
+        for _ in range(n):
+            yield val
```

Comment:
dont need this?

Reply:
oops, followed the previous comment but never deleted